src/components/autocomplete/demoFloatingLabel/index.html

```diff
@@ -16,12 +16,14 @@
             md-search-text="ctrl.searchText"
             md-items="item in ctrl.querySearch(ctrl.searchText)"
             md-item-text="item.display"
+            md-require-match=""
             md-floating-label="Favorite state">
           <md-item-template>
             <span md-highlight-text="ctrl.searchText">{{item.display}}</span>
           </md-item-template>
           <div ng-messages="searchForm.autocompleteField.$error" ng-if="searchForm.autocompleteField.$touched">
             <div ng-message="required">You <b>must</b> have a favorite state.</div>
+            <div ng-message="md-require-match">You didn't select any favorite state.</div>
```

Comment:
Looks good; thanks for the demo.

Can we just change this text to "Please select an existing state."?